.github/workflows/hrm-lambda-deploy.yml

```diff
@@ -34,6 +34,7 @@ on:
         options:
           - contact-retrieve-recording-url
           - contact-retrieve-transcript
+          - resources-import-cloudsearch
```

Comment:
Does it make sense to name this `resources-import-search` so it will work for both cloudsearch or elasticsearch depending on which way we go?